_pages/how-we-work/conferences-events-training.md

```diff
@@ -19,6 +19,8 @@ tags:
 
 TTS encourages all staff to attend conferences, speak at events and focus on professional development and training opportunities.
 
+Any "media" requests (like news or podcast interviews) are a separate process—forward the email or reach out to [press@gsa.gov](mailto:press@gsa.gov).
```

Comment:
Is there a reason media is in quotes?

Reply:
The particular case that prompted this was an interview with a (relatively small) podcast. I think of "the media" as larger news/publishing organizations, but OSC's interpretation is broader. Anyway, happy to remove if you think I should.

Reply:
I think your parenthetical indicates the broad nature of media, and the quotes around it may kind of load the phrase in some eyebrow-raising ways, especially given the current culture wars around media. But I also don't think it's a huge thing and was mostly just curious.